compiler/generator/instructions_compiler.cpp

```diff
@@ -1138,9 +1138,14 @@ ValueInst* InstructionsCompiler::generateVariableStore(Tree sig, ValueInst* exp)
                     return IB::genLoadStructVar(vname);
                 }
             } else {
-                // "Slow" variables are declared as locals in 'frame' or 'compute' functions
-                pushComputeBlockMethod(IB::genDecStackVar(vname, ctype, exp));
-                return IB::genLoadStackVar(vname);
+                if (gGlobal->gOneSample) {
```

Comment:
Why this change is needed ?

Reply:
This break the C/C++ backends. Compare the generated code in Rust and C/C++.

Reply:
when i -os and don't have -ec i use the generateControlDeclarations before the "loop" part. only when i put "slow" variable into the declare block they will be available. same for the other comment

Reply:
Change in `Instructions_compiler.cpp` may break other backends which is the case here. So try to save the issue in the Rest backend itself and do not change `Instructions_compiler.cpp`.